misc/Makefile

```diff
@@ -15,17 +17,45 @@ doc/qutebrowser.1.html:
 
 install: doc/qutebrowser.1.html
 	$(PYTHON) setup.py install --prefix="$(PREFIX)" --optimize=1 $(SETUPTOOLSOPTS)
+	$(PYTHON) setup.py install --mandir="$(datarootdir)/man" --optimize=1 $(SETUPTOOLSOPTS)
+	$(PYTHON) setup.py install --datadir="$(datarootdir)" --optimize=1 $(SETUPTOOLSOPTS)
 	install -Dm644 doc/qutebrowser.1 \
 		"$(DESTDIR)$(PREFIX)/share/man/man1/qutebrowser.1"
+	install -Dm644 doc/qutebrowser.1\
+	    "$(DESTDIR)$(datarootdir)/man/share/man/man1/qutebrowser.1"
```

Comment:
For some reason, the `dataroot` variable isn't populated when I set `MANDIR` only.

The docs aren't really clear what to do for this, but I did find this [example makefile](http://web.mit.edu/~ecprice/fuse-2.7.0/example/Makefile), which I'm guessing does things correctly (?).


So the chain appears to be:

prefix is by default `/usr/local`

datarootdir is by default `$(prefix)/share`

mandir is by default `$(datarootdir)/man`

datadir is by default `$(datarootdir)` (before it's defined??)

Maybe someone who knows makefile better than me will be able to give a better explanation :(

There's also this snippet (which dosen't explain man at all) at the bottom of the doc page)

```
# Common prefix for installation directories.
# NOTE: This directory must exist when you start the install.
prefix = /usr/local
datarootdir = $(prefix)/share
datadir = $(datarootdir)
exec_prefix = $(prefix)
# Where to put the executable for the command 'gcc'.
bindir = $(exec_prefix)/bin
# Where to put the directories used by the compiler.
libexecdir = $(exec_prefix)/libexec
# Where to put the Info files.
infodir = $(datarootdir)/info
```